SECURITY.md

```diff
@@ -0,0 +1,96 @@
+# Vulnerability Disclosure and Embargo Policy
+
+The Enarx project welcomes the responsible disclosure of vulnerabilities, including those discovered in:
+- Enarx code
+- Enarx dependencies (even compilers)
+- Enarx documentation
+- 3rd party hardware or firmware
+
+## Initial Contact
+
+All security bugs in Enarx should be reported to the security team. 
+To do so, please reach out either by email at security@enarx.dev or on the
```

Comment:
Is security@enarx.dev monitored? I sent a test note to it 2021-09-15 and did not get a reply (though my company email filters could have interfered).

Reply:
Hi @dcmiddle, I'll check if this email is getting monitored. Thanks!

Reply:
Thanks Dan for testing. Ashley has solved this now. The emails were being sent to Mike and Nathaniel's RH address.